src/react-queries/queries/opportunities.ts

```diff
@@ -22,37 +22,62 @@ import {
 import type { PortfolioAccount } from 'state/slices/portfolioSlice/portfolioSliceCommon'
 import type { AppDispatch } from 'state/store'
 
+type FetchAllArgs = {
+  dispatch: AppDispatch
+  accountId: AccountId | undefined
+  chainId: ChainId
+}
+
+const fetchAll = async ({ dispatch, accountId, chainId }: FetchAllArgs): Promise<void> => {
```

Comment:
Allows fetching without `AccountId[]` but with `ChainId[]` instead, since we won't have any AccountIds without a wallet